.babelrc

```diff
@@ -1,20 +1,20 @@
 {
-  ignore: [
-    '__mocks__',
-    '__tests__',
-    '__test_support__',
-    '/coverage',
+  "ignore": [
+    "__mocks__",
+    "__tests__",
+    "__test_support__",
+    "/coverage"
   ],
-  presets: [
-    ['env', {
-      targets: {
-        node: 6,
-      },
-    }],
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": 6
+      }
+    }]
   ],
-  env: {
-    test: {
-      ignore: null
+  "env": {
+    "test": {
+      "ignore": null
```

Comment:
Can you revert the changes to this file?

Reply:
.babelrc is json file but ok